base/linalg/bidiag.jl

```diff
@@ -154,10 +153,29 @@ for func in (:A_ldiv_B!, :Ac_ldiv_B!, :At_ldiv_B!)
         B
     end
 end
-for func in (:A_ldiv_Bt!, :Ac_ldiv_Bt!, :At_ldiv_Bt!)
+function A_ldiv_Bt!(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix)
+    mA,nA = size(A)
+    tmp = similar(B[:, 2])
```

Comment:
I think that this one might be wrong. The vector `tmp` should have length corresponding to `size(B,2)` and not `size(B,1)` as is the case now.